src/introduction-to-unified.md

```diff
@@ -0,0 +1,215 @@
+*   Know how it can be used.
+*   Know what parts (processors) you need for your (future) use case.
+*   Have a list of resources to continue learning or get started.
+
+* * *
+
+![Example of using unified](./image/unified-overview.png)
+
+**unified is a friendly interface backed by an ecosystem of plugins built for
+creating and manipulating content**. unified does this by taking markdown, HTML,
+or plain text prose, turning it into structured data, and making it available
+to over 100 plugins.  Tasks like text analysis, preprocessing, spellchecking,
+linting, and more can all be done through compatible tools, and even chained
+together.
+
+This and more is possible thanks to unified’s plugin pipeline, which lets you
+typically write one line of code to chain a feature into this process.
+It’s also possible to stitch together content from different sources
+and output it as a single source.
+
+**Bottom line: with unified, you don’t manually handle syntax or parsing.**
+
+However, the heavy lifting isn’t being done by unified itself, it’s a rather
+small module that acts as an interface to unify the handling of different
+content formats.  What makes unified powerful are the plugins that can be 
+chained to the handler of a content format.  A module that handles a certain
+content format, for instance [remark][remark] for markdown,
+is called a **processor**.
+
+unified has a family of processors, tooling, and specifications.  It’s called
+the unified collective.
+
+### unified collective
+
+The unified collective are like-minded organisations to collaboratively work
+on the innovation of content through seamless, interchangeable,
+and extendible tooling.
+
+Depending on what you want to do you will reference different organisations.
+But before getting into the examples, let’s do an introduction round!
+
+#### Processors
+
+*   [remark][remark] — markdown.
+*   [rehype][rehype] — HTML.
+*   [retext][retext] — natural language.
+*   [redot][redot] — Graphviz.
+
+There is also a new markdown parser in the works, [micromark][micromark]!
+
+#### Tools
+
+*   [mdx][mdx] — JSX in Markdown.
+*   [vfile][vfile] — Virtual file format for text processing.
+
+#### Specifications
+
+*   [unist][unist] — Universal Syntax Tree.
+*   [mdast][mdast] — Markdown Abstract Syntax Tree format.
+*   [hast][hast] — HTML Abstract Syntax Tree format.
+*   [nlcst][nlcst] — Natural Language Concrete Syntax Tree format.
+
+* * *
+
+Don’t worry, we’ll get to how all of this comes together in unified.
+But if you are already feeling adventurous; you can go directly to
+[“Using unified”](<>) or [“How to get started with plugins”](<>).
+
+* * *
+
+### How does it all come together
+
+unified can integrate with the file-system through
+[unified-engine][unified-engine].  On top of that, CLI apps can be created with
+[unified-args][unified-args], Gulp plugins with
+[unified-engine-gulp][unified-engine-gulp], and Atom Linters with
+[unified-engine-atom][unified-engine-atom].
```

Comment:
I think the contents of this paragraph is unnecessarily detailed for new people. Drop it or move it down?